Rename NonGeographicalRegion to NonGeographicalEntity.

KarolJakubKrawiec opened the pull request:
This is to be consistent with the naming of non-geographical entities, since region and non-geographical can be contradictory.

Context: b/206752240

KarolJakubKrawiec committed:
Rename NonGeographicalRegion to NonGeographicalEntity.

This is to be consistent with the naming of non-geographical entities, since region and non-geographical can be contradictory.

tvislavski reviewed (commented):
Can you also update https://github.com/google/libphonenumber/blob/master/pending_code_changes.txt with the explanation of this change? This will be included in the release notes then

KarolJakubKrawiec committed:
Modify pending_code_changes.txt

Added ( - Renamed NonGeographicalRegion to NonGeographicalEntity.
 - Created a new interface called NonGeographicalEntityMetadataSourceV2) into the file.

KarolJakubKrawiec committed:
Reform @deprecated anotation

Changed @deprecated use the following... to just @deprecated Use... to make every @deprecated comment the same.

KarolJakubKrawiec committed:
Rename NonGeographicalRegion to NonGeographicalEntity

I renamed the NonGeographicalRegion variables and methodes to NonGeographicalEntity because we want to keep the naming the same so its easier to read. This CL is for the C++ version.

KarolJakubKrawiec committed:
finish renaming NonGeoRegions to NonGeoEntities

KarolJakubKrawiec committed:
Rewrote renaming for geographicalEntity

KarolJakubKrawiec committed:
Rename Region to Entity in GetMetadataForNonGeographicalEntity

KarolJakubKrawiec committed:
Merge branch 'google:master' into rename-nongeographicalregion

silvio2402 commented on cpp/src/phonenumbers/phonenumberutil.h:
```diff
@@ -865,6 +865,9 @@ class PhoneNumberUtil : public Singleton<PhoneNumberUtil> {
   const i18n::phonenumbers::PhoneMetadata* GetMetadataForNonGeographicalRegion(
       int country_calling_code) const;
```
You can remove this since it is a private function.

silvio2402 commented on cpp/test/phonenumbers/phonenumberutil_test.cc:
```diff
@@ -66,7 +66,12 @@ class PhoneNumberUtilTest : public testing::Test {
 
   const PhoneMetadata* GetMetadataForNonGeographicalRegion(
       int country_code) const {
-    return phone_util_.GetMetadataForNonGeographicalRegion(country_code);
+    return phone_util_.GetMetadataForNonGeographicalEntity(country_code);
+  }
```
You can also remove this because you already renamed the calls to this function.

KarolJakubKrawiec committed:
Merge branch 'google:master' into rename-nongeographicalregion

KarolJakubKrawiec committed:
Correct mistakes in pull request

In this commit I have delete unnecessary code.

KarolJakubKrawiec committed:
Merge branch 'google:master' into rename-nongeographicalregion

silvio2402 reviewed (approved):
LGTM

mandlil committed:
Merge branch 'master' into rename-nongeographicalregion
